site/src/components/navbar.tsx

```diff
@@ -151,12 +158,12 @@ const useScrollingNavbar = (
 export const Navbar: FunctionComponent<NavbarProps> = ({ openLoginModal }) => {
   const theme = useTheme();
   const { pathname } = useRouter();
-  const asPath = useHydrationFriendlyAsPath();
+  const hydrationFriendlyAsPath = useHydrationFriendlyAsPath();
```

Comment:
I’ve renamed `asPath` to `hydrationFriendlyAsPath` to avoid confusion between the two. I was still using real `asPath` in mobile navbar components, which kept triggering warnings like this:

```
Warning: Prop `href` did not match. Server: "/docs" Client: "/docs#"
```

We don't have to pass `hydrationFriendlyAsPath` as a prop because `useHydrationFriendlyAsPath()` can be called everywhere in theory. I still use a prop to avoid extra re-renders because of `useEffect` inside each `useHydrationFriendlyAsPath()`. Values may go out of sync with the current hook implementation, which can trigger warnings or inconsistent renders.